MarkBind/markbind
# Allow navbar theming

**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [X] Enhancement to an existing feature

Fixes #386


**What is the rationale for this request?**
This PR allows users to customize their navbar colors. There is now an option to choose from 3 color schemes that are provided by each bootstrap/bootswatch theme.

There's also an option to specify `type="none"`, for users who wish to inject their own CSS classes using our existing `add-class` feature.

In short, `<navbar>` now has `type="primary" | "dark" | "light" | "none"`, instead of `type="inverse" | "default"`.

**What changes did you make? (Give an overview)**
- https://github.com/MarkBind/vue-strap/pull/102
- Updated all instances in docs and tests where we used a `<navbar>`
- Other minor cleanup


**Testing instructions:**
1. Look at `/userGuide/usingComponents.html#navbars` for the code and the corresponding output.
2. Change the navbar type to `none` or some gibberish value (this will default to `primary`), observe that the rendered navbar is styled correctly.
